`@slicemachine/manager`: peerDependency of `msw@1` causes a conflict in projects using `msw@2`

<!-- 💙 Thanks for your time to make this package better with your feedback 💙

**IMPORTANT** Before reporting a bug please make sure that you have read through the documentation:
- https://prismic.io/docs

👍 A properly detailed bug report can save a LOT of time and help fixing issues as soon as possible.
-->

### Versions

- slice-machine-ui: 2.18.2
- node: v22.20.0

### When did this happen?

October 23, 2025, 2:14PM CST

### Reproduction


<!-- Link to a minimal test case, without a reproduction, it is so hard to address problem :( -->

<details open>
<summary>Additional Details</summary>
<br>
<!-- Attaching `package.json`, dependencies, logs or code snippets would help to find the issue -->
</details>

### Steps to reproduce

Just run the following:

```sh

npm init -y

npm install msw                       # latest version: 2.11.6
npm install @slicemachine/manager     # only lists v1 as a peerDependency
```

### What is expected?

Should install `@slicemachine/manager`

### What is actually happening?

Fails to install without a `--legacy-peer-deps` flag with the following error:

```
❯ npm install -D @slicemachine/manager
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @slicemachine/manager@0.25.5
npm error Found: msw@2.11.6
npm error node_modules/msw
npm error   dev msw@"^2.11.6" from the root project
npm error
npm error Could not resolve dependency:
npm error peerOptional msw@"^1.1.0" from @slicemachine/manager@0.25.5
npm error node_modules/@slicemachine/manager
npm error   dev @slicemachine/manager@"*" from the root project
npm error
npm error Conflicting peer dependency: msw@1.3.5
npm error node_modules/msw
npm error   peerOptional msw@"^1.1.0" from @slicemachine/manager@0.25.5
npm error   node_modules/@slicemachine/manager
npm error     dev @slicemachine/manager@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\josephh\AppData\Local\npm-cache\_logs\2025-10-23T19_26_26_919Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\josephh\AppData\Local\npm-cache\_logs\2025-10-23T19_26_26_919Z-debug-0.log
```

I fixed this by adding an override in our package.json for now:

```json
{
  "overrides": {
    "@slicemachine/manager": {
      "msw": "^2.11.3"
    }
  }
}
```

But it seems like `@slicemachine/manager` could either remove msw from its peerDependencies or update it to the following:

```json
{
  "peerDependencies": {
    "msw": ">=1.1.0"
  }
}
```

--- a/packages/manager/package.json
+++ b/packages/manager/package.json
@@ -118,7 +118,7 @@
 		"vitest": "0.32.0"
 	},
 	"peerDependencies": {
-		"msw": "^1.1.0"
+		"msw": ">=1.1.0"
 	},
 	"peerDependenciesMeta": {
 		"msw": {

--- a/packages/manager/package.json
+++ b/packages/manager/package.json
@@ -117,6 +117,14 @@
 		"vite-plugin-sdk": "0.1.1",
 		"vitest": "0.32.0"
 	},
+	"peerDependencies": {
+		"msw": "^1.1.0"
+	},
+	"peerDependenciesMeta": {
+		"msw": {
+			"optional": true
+		}
+	},
 	"engines": {
 		"node": ">=14.15.0"
 	},

--- a/packages/manager/package.json
+++ b/packages/manager/package.json
@@ -117,14 +117,6 @@
 		"vite-plugin-sdk": "0.1.1",
 		"vitest": "0.32.0"
 	},
-	"peerDependencies": {
-		"msw": "^1.1.0"
-	},
-	"peerDependenciesMeta": {
-		"msw": {
-			"optional": true
-		}
-	},
 	"engines": {
 		"node": ">=14.15.0"
 	},

